sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -235,6 +235,23 @@ object Project {
   }
 }
 
+case class UnresolvedDropColumns(dropList: Seq[Expression], child: LogicalPlan)
```

Comment:
name it `UnresolvedDropColumns` to avoid conflict with `DropColumns` in `v2AlterTableCommands`

Reply:
how about `DataFrameDropColumns`?

Reply:
sounds good